ci/jenkins/Jenkins_steps.groovy

```diff
@@ -174,7 +174,7 @@ def compile_unix_mkl_cpu(lib_name) {
 }
 
 def compile_unix_onednn_cpu(lib_name) {
-    return ['CPU: ONEDNN': {
+    return ['CPU: oneDNN': {
```

Comment:
please double check if "oneDNN" here works properly

Reply:
Works properly, this just changes name of build/test scenario visible on Jenkins e.g. https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-20719/3/pipeline.